sky/unit/test/engine/mock_events.dart

```diff
@@ -79,3 +81,71 @@ class TestGestureEvent extends sky.GestureEvent {
   double velocityX;
   double velocityY;
 }
+
+class _PointerState {
+  bool isDown = false;
+  sky.Point location;
+}
+
+class TestPointerGenerator {
```

Comment:
I thought about doing that, but I was worried about state leaking between tests.  Maybe we should just make the _nextId a member variable of the generator rather than a static.  Each test should new up a different generator, so that will isolate the state between the tests.